cluster-install/install.sh

```diff
@@ -216,13 +217,13 @@ load_config_from_file_if_given() {
 
     # shellcheck disable=SC1090
     source "$config_file"
+    set_log_level
```

Comment:
This fixes an issue where the `logwarn` right below would fail (and output a bunch of errors) because the log level wasn't set yet.